source/sphinx_extensions/plugin_directory/extension.py

```diff
@@ -84,8 +84,12 @@ def generate_rst(app):
                 template = env.get_template('action.rst')
                 rendered = template.render(title=title, cli_help=cli_help,
                                            api_help=api_help, bib_id=bib_id,
-                                           has_citations=bool(action.citations)
-                                           )
+                                           has_citations=bool(
+                                            action.citations),
+                                           plugin_cli_name=plugin_cli_name,
+                                           action_cli_name=action_cli_name,
+                                           plugin_api_name=plugin.name,
+                                           action_api_name=action.id)
```

Comment:
Unfortunately this won't exactly work. The reason is that `plugin.name` will be something like `"feature-table"` whereas the Python API will see `feature_table`. You could use `.replace('-', '_')` but the better way to do this would be to call `action.get_import_path()` and then pop off the last segment.